ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -440,8 +445,12 @@ QtObject {
         walletSection.runEditAccountPopup(address)
     }
 
-    function switchReceiveAccountByAddress(address) {
-        walletSectionSend.switchReceiveAccountByAddress(address)
+    function setSenderAccount(address) {
+        walletSectionSend.setSenderAccount(address)
```

Comment:
The biggest question in this PR for is why this currently selected sender account must be set to the backend. Is there a good reason to do that this way or it's only kind of holding dirty state of UI on the nim side?

Reply:
Please create task for that and link with this issue :pray: